tests/lib/rules/attribute-hyphenation.js

```diff
@@ -58,6 +58,16 @@ ruleTester.run('attribute-hyphenation', rule, {
       filename: 'test.vue',
       code: '<template><div><slot myProp></slot></div></template>',
       options: ['never']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div><custom :attr_ff="prop"></custom></div></template>',
+      options: ['always']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div><custom :_attr_ii="prop"></custom></div></template>',
+      options: ['always']
     }
```

Comment:
What is the difference between these test cases? I think one of them can be dropped.